core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/codeGen/CodeWithConverter.kt

```diff
@@ -28,3 +25,6 @@ public data class CodeWithConverter(val declarations: Code, val converter: (Vari
             else -> declarations + "\n" + converter(name)
         }
 }
+
+public typealias Code = String
```

Comment:
Is it an option to not have these in public api?

Reply:
I think they're part of our codegen api. CodeWithConverter is public too after all